README.md

```diff
@@ -110,3 +110,39 @@ through Django shell (`./compose run --rm api manage shell`), use the following
 >>> user.groups.add(pe_group)
 >>> user.save()
 ```
+
+## Pulp 3 Installer
+
+#### 1. Clone the GitHub repos
+
+```console
+$ git clone https://github.com/ansible/galaxy_ng.git
+$ git clone https://github.com/pulp/pulplift.git
+```
+
+#### 2. Setup pulplift
```

Comment:
The requirements need to be installed. We cannot assume users have them.

https://github.com/pulp/pulplift#requirements